yaptide/utils.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime, timedelta
+from typing import Union
+from secrets import token_hex
+
+import jwt
+
+SECRET_KEY_TOKEN = token_hex(64)
+SECRET_KEY_TOKEN_REFRESH = token_hex(64)
+_Refresh_Token_Expiration_Time = 120
+_Access_Token_Expiration_Time = 10
+
+
+def encode_auth_token(user_id: int, isRefresh: bool = False) -> tuple[Union[str, Exception], datetime]:
+    """Function encoding the token"""
+    if isRefresh:
+        secret = SECRET_KEY_TOKEN_REFRESH
+        exp_time_minutes = _Refresh_Token_Expiration_Time
+    else:
+        secret = SECRET_KEY_TOKEN
+        exp_time_minutes = _Access_Token_Expiration_Time
+    exp = datetime.utcnow() + timedelta(minutes=exp_time_minutes)
+    try:
+        payload = {
+            'exp': exp,
+            'iat': datetime.utcnow(),
```

Comment:
what is `iat` ? I cannot assign any meaning to that variable

Reply:
Issued At Time

Reply:
![image](https://user-images.githubusercontent.com/56313631/143411131-4420f3a4-0e76-4580-a8c9-8ff0de243cff.png)

Reply:
Can you add a link to that table in a comment ? Together with longer name explanation ?

Reply:
https://en.wikipedia.org/wiki/JSON_Web_Token

Reply:
```suggestion
            'iat': datetime.utcnow(),  # Issued At
```